src/WorkerManager.ts

```diff
@@ -0,0 +1,46 @@
+/*
+Copyright 2022 The Matrix.org Foundation C.I.C.
```

Comment:
Super minor - think this should be 2023. Also same goes for `playback.worker.ts` and `worker.ts`.

Reply:
The code was written in 2022, the PR was just a draft for ages